### ISSUE TITLE
Submitting a PIN

### ISSUE THREAD

**Issue by mnataraj92** [episphere/connect#729]
Feedback from Nina at UC regarding submitting a PIN:
"The 'I do not have a PIN' button is immediately under PIN text box. The button location and color vibrancy make it easy to accidently select after entering a PIN. The ”Submit” button that we want people to select is much further away and in a more muted color, making it less intuitive to see and select."

Is there a way we can make the screen more intuitive for hitting 'Submit' as opposed to 'I do not have a PIN'?

Nina also provided a visual for a potential solution (attached)
[PIN Submit Button.pdf](https://github.com/episphere/connectApp/files/11152647/PIN.Submit.Button.pdf)

**Comment by robertsamm** [comment]
I don't think this is something we want to change at this point in time. @brotzmanmj if you agree, we can close this issue.

**Comment by brotzmanmj** [comment]
The suggestion is actually a good one and looks like a relatively straightforward fix. Let's keep this one and see if we can fix it over the summer.

**Comment by robertsamm** [comment]
Sounds good, assigning it an August milestone.

**Comment by depietrodeanna** [comment]
@robertsamm @brotzmanmj I agree this would be a good adjustment. 

**Notes for DevOps:** 

The "Submit" button should be the colors it normally has throughout the user profile (yellow background with blue wording)

The "I do not have a PIN" button should remain as is, blue background with white text.

Here is the language I suggest for the content above the "I do not have a PIN"
"If you don't have a PIN, click the button below to continue."

This is the mockup that shows how we'd like the layout to look, disregard the text in these photos and use what I shared above:

<img width="525" height="306" alt="Image" src="https://github.com/user-attachments/assets/cf4c78dd-6b71-42d0-86cc-61744215462f" />

**Comment by robertsamm** [comment]
Hello @we-ai this looks great to me! Thanks!

**Comment by we-ai** [comment]
> Hello [@we-ai](https://github.com/we-ai) this looks great to me! Thanks!

Hi @robertsamm, thanks for your feedback. 
Additional change: now the input box is slightly shorter.

**Comment by robertsamm** [comment]
Noting that @depietrodeanna will lead the testing on this but can loop me in if needed

**Comment by depietrodeanna** [comment]
@we-ai I am ready to test this in Dev when it is time.

**Comment by depietrodeanna** [comment]
tested in Dev. Ready for Stage @we-ai @sonyekere
